clock/test_clock_test.go

```diff
@@ -61,3 +62,20 @@ func TestTickAfter(t *testing.T) {
 	tickOrTimeOut(ticker2, true)
 	tickOrTimeOut(ticker3, false)
 }
+
+func TestTickAfterConcurrent(t *testing.T) {
```

Comment:
Maybe rename to something that highlights the signal? The old ticker was also 'concurrent'

Reply:
done